### PR TITLE
[4.0] scss lint testing

### PR DESCRIPTION
**Pull request by brianteeman**
testing the hound
working through the scss to get it back in shape

### CODE REVIEW

**Comment by brianteeman** [comment]
@c-lodder seems to be working well now - I just need to re-enable the ordering rule and work through those

File: installation/template/scss/template.scss [hunk 1]
```diff
@@ -230,18 +230,18 @@ caption {
   color: #292b2c;
 }
 
-.fa-eye:before,
-.icon-eye-open:before,
-.icon-eye:before {
+.fa-ey,
```
Comment (C-Lodder) [inline installation/template/scss/template.scss]: You've removed the last letter from each class here

Comment (brianteeman) [reply installation/template/scss/template.scss]: no idea how that happened

**Comment by brianteeman** [comment]
@c-lodder @ciar4n After going through everything locally with the lint I am left with the issues you can see above and the following. Your advice please - I suspect some might be rule changes

```


administrator/components/com_media/resources/styles/components/_media-breadcrumb.scss:16:1 [W] UnnecessaryParentReference: Unnecessary parent selector (&)
administrator/components/com_media/resources/styles/components/_media-edit.scss:1:1 [W] IdSelector: Avoid using id selectors
administrator/components/com_media/resources/styles/components/_media-modal.scss:67:3 [W] PropertySortOrder: Properties should be ordered position, top, right, margin, font-size, color, background, border, opacity
administrator/templates/atum/scss/blocks/_global.scss:184:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/blocks/_global.scss:206:3 [W] PropertyUnits: fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
administrator/templates/atum/scss/blocks/_global.scss:209:5 [W] PropertyUnits: fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
administrator/templates/atum/scss/blocks/_global.scss:209:5 [W] PropertyUnits: fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
administrator/templates/atum/scss/blocks/_global.scss:218:7 [W] PropertyUnits: fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
administrator/templates/atum/scss/blocks/_global.scss:218:7 [W] PropertyUnits: fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
administrator/templates/atum/scss/blocks/_global.scss:218:7 [W] PropertyUnits: fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
administrator/templates/atum/scss/blocks/_header.scss:44:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_header.scss:48:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_header.scss:52:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_header.scss:123:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_header.scss:131:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_header.scss:147:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_header.scss:234:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_header.scss:234:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_header.scss:238:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_header.scss:238:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_header.scss:264:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/blocks/_header.scss:352:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/blocks/_header.scss:374:1 [W] UnnecessaryParentReference: Unnecessary parent selector (&)
administrator/templates/atum/scss/blocks/_header.scss:374:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_header.scss:377:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_header.scss:384:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_layout.scss:42:2 [W] Comment: Use `//` comments everywhere
administrator/templates/atum/scss/blocks/_lists.scss:1:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/blocks/_login.scss:15:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/blocks/_login.scss:63:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/blocks/_login.scss:69:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_login.scss:69:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_login.scss:80:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/blocks/_login.scss:105:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_login.scss:140:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/blocks/_login.scss:146:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/blocks/_login.scss:158:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/blocks/_login.scss:221:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/blocks/_login.scss:225:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/blocks/_login.scss:225:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/blocks/_login.scss:232:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/blocks/_login.scss:238:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/blocks/_quickicons.scss:22:7 [W] VendorPrefix: Avoid vendor prefixes.
administrator/templates/atum/scss/blocks/_quickicons.scss:40:9 [W] VendorPrefix: Avoid vendor prefixes.
administrator/templates/atum/scss/blocks/_quickicons.scss:145:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_quickicons.scss:147:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_quickicons.scss:166:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_quickicons.scss:201:11 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_quickicons.scss:216:9 [W] VendorPrefix: Avoid vendor prefixes.
administrator/templates/atum/scss/blocks/_quickicons.scss:227:11 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_quickicons.scss:230:13 [W] VendorPrefix: Avoid vendor prefixes.
administrator/templates/atum/scss/blocks/_quickicons.scss:246:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/blocks/_quickicons.scss:260:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/blocks/_sidebar.scss:144:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:144:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:151:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:157:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:173:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:173:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:174:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:176:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:210:5 [W] EmptyRule: Empty rule
administrator/templates/atum/scss/blocks/_sidebar.scss:225:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:233:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:254:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:262:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:262:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:281:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:308:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:308:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:312:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:312:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:320:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:320:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:324:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:324:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_sidebar.scss:512:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/blocks/_sidebar.scss:555:7 [W] Comment: Use `//` comments everywhere
administrator/templates/atum/scss/blocks/_sidebar.scss:567:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/blocks/_sidebar.scss:578:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/blocks/_sidebar.scss:587:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/blocks/_toolbar.scss:63:5 [W] DuplicateProperty: Property `box-shadow` already defined on line 62
administrator/templates/atum/scss/blocks/_toolbar.scss:96:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_toolbar.scss:125:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_toolbar.scss:158:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_toolbar.scss:179:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_toolbar.scss:205:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_toolbar.scss:233:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_toolbar.scss:265:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/blocks/_treeselect.scss:123:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/pages/_com_config.scss:5:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/pages/_com_config.scss:6:5 [W] PropertyUnits: fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
administrator/templates/atum/scss/pages/_com_config.scss:9:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/pages/_com_config.scss:28:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/pages/_com_content.scss:9:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/pages/_com_content.scss:9:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/pages/_com_content.scss:14:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/pages/_com_content.scss:26:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/pages/_com_tags.scss:3:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/pages/_com_tags.scss:3:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/pages/_com_tags.scss:6:7 [W] PropertyUnits: fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, 
em, ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
administrator/templates/atum/scss/pages/_com_tags.scss:15:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/pages/_com_tags.scss:19:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/pages/_com_users.scss:24:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/system/searchtools/searchtools.scss:86:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/system/searchtools/searchtools.scss:240:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/system/searchtools/searchtools.scss:240:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/system/searchtools/searchtools.scss:246:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/system/searchtools/searchtools.scss:246:11 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/system/searchtools/searchtools.scss:250:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/system/searchtools/searchtools.scss:250:11 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/system/searchtools/searchtools.scss:255:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/system/searchtools/searchtools.scss:268:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/system/searchtools/searchtools.scss:268:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/template-rtl.scss:187:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/template-rtl.scss:251:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/template-rtl.scss:251:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/template-rtl.scss:251:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/template-rtl.scss:258:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/vendor/bootstrap/_dropdown.scss:49:1 [W] UnnecessaryParentReference: Unnecessary parent selector (&)
administrator/templates/atum/scss/vendor/bootstrap/_table.scss:30:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/vendor/choicesjs/choices.scss:23:3 [W] DuplicateProperty: Property `border-radius` already defined on line 22
administrator/templates/atum/scss/vendor/choicesjs/choices.scss:106:5 [W] UrlQuotes: URLs should be enclosed in quotes
administrator/templates/atum/scss/vendor/choicesjs/choices.scss:111:7 [W] UrlQuotes: URLs should be enclosed in quotes
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-alert.scss:14:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-alert.scss:134:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss:63:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss:88:5 [W] DuplicateProperty: Property `border` already defined on line 85
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss:142:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss:221:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss:240:1 [W] IdSelector: Avoid using id selectors
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss:245:11 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss:246:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss:250:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5   
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss:255:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5   
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss:255:11 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss:259:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5   
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss:259:11 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss:265:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5   
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss:265:11 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss:269:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 6   
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss:274:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 6   
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss:280:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5   
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss:280:11 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss:284:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5   
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss:284:11 [W] NestingDepth: Nesting should be no greater than 4, but was 5
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss:288:11 [W] NestingDepth: Nesting should be no greater than 4, but was 5
build/media_source/plg_editors_tinymce/scss/tinymce-builder.scss:1:1 [W] IdSelector: Avoid using id selectors
installation/template/scss/template-rtl.scss:8:1 [W] IdSelector: Avoid using id selectors
installation/template/scss/template-rtl.scss:86:1 [W] IdSelector: Avoid using id selectors
templates/cassiopeia/scss/blocks/_css-grid.scss:6:5 [W] SpaceAfterPropertyColon: Colon after property should be followed by at least one space
templates/cassiopeia/scss/blocks/_css-grid.scss:51:5 [W] DuplicateProperty: Property `grid-template-areas` already defined on line 6
templates/cassiopeia/scss/blocks/_css-grid.scss:64:7 [W] SpaceAfterPropertyColon: Colon after property should be followed by at least one space
templates/cassiopeia/scss/blocks/_css-grid.scss:152:3 [W] PropertyUnits: fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
templates/cassiopeia/scss/blocks/_css-grid.scss:159:5 [W] PropertyUnits: fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
templates/cassiopeia/scss/blocks/_css-grid.scss:159:5 [W] PropertyUnits: fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
templates/cassiopeia/scss/blocks/_header.scss:24:14 [W] Shorthand: Shorthand form for property `padding` should be written more concisely as `2.5rem $cassiopeia-grid-gutter / 2` instead of `2.5rem $cassiopeia-grid-gutter / 2 2.5rem`
templates/cassiopeia/scss/blocks/_modifiers.scss:108:12 [W] Shorthand: Shorthand form for property `padding` should be written more concisely as `0 $cassiopeia-grid-gutter / 2 $cassiopeia-grid-gutter` instead of `0 $cassiopeia-grid-gutter / 2 $cassiopeia-grid-gutter $cassiopeia-grid-gutter / 2`
templates/cassiopeia/scss/blocks/_modifiers.scss:168:5 [W] PropertyUnits: fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, 
ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
templates/cassiopeia/scss/blocks/_modifiers.scss:186:9 [W] PropertyUnits: fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, 
ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
templates/cassiopeia/scss/blocks/_modifiers.scss:186:9 [W] PropertyUnits: fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, 
ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
templates/cassiopeia/scss/blocks/_modifiers.scss:190:9 [W] PropertyUnits: fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, 
ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
templates/cassiopeia/scss/blocks/_modifiers.scss:190:9 [W] PropertyUnits: fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, 
ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
templates/cassiopeia/scss/blocks/_modifiers.scss:190:9 [W] PropertyUnits: fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, 
ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
templates/cassiopeia/scss/blocks/_modifiers.scss:194:9 [W] PropertyUnits: fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, 
ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
templates/cassiopeia/scss/blocks/_modifiers.scss:194:9 [W] PropertyUnits: fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, 
ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
templates/cassiopeia/scss/blocks/_modifiers.scss:194:9 [W] PropertyUnits: fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, 
ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
templates/cassiopeia/scss/blocks/_modifiers.scss:194:9 [W] PropertyUnits: fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, 
ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
templates/cassiopeia/scss/vendor/_chosen.scss:97:9 [W] NestingDepth: Nesting should be no greater than 4, but was 5

```

File: templates/cassiopeia/scss/blocks/_modifiers.scss [hunk 2]
```diff
@@ -103,13 +103,13 @@
 }
 
 .blog-item {
-  padding: 0 ($cassiopeia-grid-gutter / 2) $cassiopeia-grid-gutter ($cassiopeia-grid-gutter / 2);
   display: flex;
   flex-direction: column;
+  padding: 0 ($cassiopeia-grid-gutter / 2) $cassiopeia-grid-gutter ($cassiopeia-grid-gutter / 2);
```
Comment (C-Lodder) [inline templates/cassiopeia/scss/blocks/_modifiers.scss]: Replace with `padding: 0 ($cassiopeia-grid-gutter / 2) $cassiopeia-grid-gutter`

File: administrator/templates/atum/scss/blocks/_header.scss [hunk 3]
```diff
@@ -50,17 +50,18 @@
           display: inline-block;
         }
 
-        path
-        {fill: $white}
+        path {
```
[the remark this answers is not in the document]
Comment (C-Lodder) [reply administrator/templates/atum/scss/blocks/_header.scss:52]: This should be removed anyway. I've mentioned that on another issue

**Comment by C-Lodder** [comment]
@brianteeman

Courtesy of unreviewed PRs being merged.

I'm not sure about the grid errors. Perhaps scss-lint doesn't properly support all the grid properties yet.

As for nested being greater that 4 levels and ID selectors....these should be fixed. Id selectors are too tightly coupled with the HTML. They should only be used in very specific cases when necessary.

**Comment by C-Lodder** [comment]
@brianteeman

Courtesy of unreviewed PRs being merged.

I'm not sure about the grid errors. Perhaps scss-lint doesn't properly support all the grid properties yet.

As for nested being greated that 4 levels and ID selectors....these should be fixed. Id selectors are too tightly coupled with the HTML. They should only be used in very specific cases when necessary.

**Comment by brianteeman** [comment]
Maybe @wilsonge can merge this now. Before it gets in merge conflict hell and then I can create the issues for the rest.

**Comment by brianteeman** [comment]
Maybe @rdeutz can merge this now. Before it gets in merge conflict hell and then I can create the issues for the rest.

**Comment by brianteeman** [comment]
When #28083 is merged another big batch of issues will go away ;)

**Comment by ciar4n** [comment]
As a rule, you would often see IDs only used if required by JS. 

Excessive nesting levels in some cases might be a tricky fix. It really depends if the nesting was done because it was 'easy' or had the specific purpose to increase precedence. Either way, probably the hardest fix on the list.

**Comment by brianteeman** [comment]
Yes all of those are beyond my skillset but if we merge this as is it will make it much easier to keep everything clean in the future and to more easily revue outstanding issues

File: templates/cassiopeia/scss/blocks/_modifiers.scss [hunk 4]
```diff
@@ -163,10 +163,10 @@
 @supports (display: grid) {
   .blog-items {
     display: grid;
-    grid-gap: $cassiopeia-grid-gutter;
-    grid-template-columns: 1fr;
-    grid-auto-flow: row;
     margin: 0 0 $cassiopeia-grid-gutter;
+    grid-auto-flow: row;
+    grid-template-columns: 1fr;
```
[the remark this answers is not in the document]
Comment (brianteeman) [reply templates/cassiopeia/scss/blocks/_modifiers.scss:168]: resolved by #28083

**Comment by ciar4n** [comment]
I have tested this item :white_check_mark: successfully on 44e65e2639dfa16d7bc962d5fc2fc1edc8a46a63<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28076">issues.joomla.org/tracker/joomla-cms/28076</a>.</sub>

File: templates/cassiopeia/scss/blocks/_modifiers.scss [hunk 5]
```diff
@@ -163,10 +163,10 @@
...
+    grid-gap: $cassiopeia-grid-gutter;
```
[the remark this answers is not in the document]
Comment (brianteeman) [reply templates/cassiopeia/scss/blocks/_modifiers.scss:169]: resolved by #28082

**Comment by C-Lodder** [comment]
I have tested this item :white_check_mark: successfully on 44e65e2639dfa16d7bc962d5fc2fc1edc8a46a63<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28076">issues.joomla.org/tracker/joomla-cms/28076</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
woot i can retire now

**Pull request merged**